Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM python:2.7
 ADD validate-vrt.py /
```

Comment:
Keep in mind that the `ADD` and `COPY` directives actually bake the files you specify into the image – it's probably better to specify in a Docker section within CONTRIBUTING.md that the container should be run with something like `-v .:/tmp/vrt` as an option with `docker run`. That will mount the current local directory (with all of the files you're using `ADD` for now) inside the container at runtime.